prep-javascript-variables/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    const fullName = 'Eric Do';
+    console.log(fullName);
+    const numberOfPets = '3';
```

Comment:
For numbers, we'll want to leave off the quotes so that the javascript engine doesn't interpret them as strings

Reply:
Got it, I think I submitted the changes correctly. Let me know if I did not, thanks.  I will attach photos here as well just in case

![prep-JS-variables-changed](https://user-images.githubusercontent.com/72995114/203433884-f0413fa6-5a97-41f2-a9a4-3c095
![prep-JS-variables-code-change](https://user-images.githubusercontent.com/72995114/203433890-466c831b-3fef-4612-b897-cba1efc64541.png)
d1d8628.png)